File: data/skills/clawhub_ai-hub-admin__generate-perler-bead-pindou/files/scripts/generate_perler_bead_pindou.js
#!/usr/bin/env node

const { spawnSync } = require("node:child_process");
const path = require("node:path");

const script = path.join(__dirname, "run_api.js");
const args = process.argv.slice(2);
const res = spawnSync(process.execPath, [script, "--api", "generate_perler_bead_pindou", ...args], { stdio: "inherit" });
process.exit(typeof res.status === "number" ? res.status : 1);

File: data/skills/clawhub_ai-hub-admin__generate-perler-bead-pindou/files/scripts/generate_perler_bead_pindou.py
#!/usr/bin/env python3

import os
import subprocess
import sys

def main():
    script = os.path.join(os.path.dirname(__file__), "run_api.py")
    args = [sys.executable, script, "--api", "generate_perler_bead_pindou"] + sys.argv[1:]
    res = subprocess.run(args)
    raise SystemExit(res.returncode)

if __name__ == "__main__":
    main()
